File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float total;
    int coins = 0;

    do
    {
        total = get_float("Change owed: ");
    }
    while (total < 0);
    total = round(total * 100);


    for (int i = 0; total >= 25; i++)
    {
        total = total - 25;
        coins++;
    }
    for (int i = 0; total >= 10; i++)
    {
        total = total - 10;
        coins++;
    }
    for (int i = 0; total >= 5; i++)
    {
        total = total - 5;
        coins++;
    }
    for (int i = 0; total >= 1; i++)
    {
        total = total - 1;
        coins++;
    }


    printf("%i\n", coins);

}

File: pset6/cash/cash.py
from cs50 import get_float

# Take input from user
while True:
    change = get_float("Change owed: ")*100
    if change > 0:
        break

coin = 0
# Calculate the change 
# Calculate the change 
# Calculate the change 
# Calculate the change 
while(change >= 25):
    change = change-25
    coin += 1

while(change >= 10):
    change = change-10
    coin += 1

while(change >= 5):
    change = change-5
    coin += 1

while(change >= 1):
    change = change-1
    coin += 1

print(coin)